### PR TITLE
Apply copy editor changes to security.md

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: book/security.md [hunk 1]
```diff
@@ -433,23 +431,18 @@ Cross-site requests
...
+server's cookies.[^tls] But if an attacker is
 clever, they might be able to get *the server* or *the browser* to
 help them steal cookie values.
 
 [^tls]: Well... Our connection isn't encrypted, so an attacker could
     read it from an open Wifi connection. But another *server*
-    couldn't.
-
-[^dns]: Well... Another server could hijack our DNS and redirect our
+    couldn't. Or, another server could hijack our DNS and redirect our
```
Comment (pavpanchekha) [inline book/security.md:440]:
```suggestion
    couldn't. Though, another server could hijack our DNS and redirect our
```

File: book/security.md [hunk 2]
```diff
@@ -433,23 +431,18 @@ Cross-site requests
...
     hostname to a different IP address, and then steal our cookies.
     Some ISPs support DNSSEC, which prevents this, but not all.
-
-[^bgp]: Well... A state-level attacker could announce fradulent BGP
+    Or, a state-level attacker could announce fradulent BGP
     routes, which would send even a correctly-retrieved IP address to
-    the wrong physical computer.
-
-[^oof]: Security is very hard.
+    the wrong physical computer. (Security is very hard.)
```
Comment (pavpanchekha) [inline book/security.md:445]: I'm sure this sequence of four footnotes violates Oxford's house style but I do think refactoring it all into one footnote loses a bit of the punch and starts to instead look like an irrelevant tangent.

File: book/security.md [hunk 3]
```diff
@@ -1007,29 +1000,25 @@ The server would then output this HTML:
...
-     whole page with a link that goes to their site and includes the
-     token value in the URL. You've seen "please click to continue"
-     screens and have clicked through unthinkingly; your users will
-     too.
+    trigger additional requests. In our limited browser the attack has to
+    be a little clunkier, but the evil script still can, for example,
+    replace the whole page with a link that goes to their site and
+    includes the token value in the URL. You've seen "please click to
+    continue" screens and have clicked through unthinkingly; your users
+    will too.
```
Comment (pavpanchekha) [inline book/security.md:1019]: Whereas this multi-footnote-refactor is, I think, totally fine.

**Review by pavpanchekha — APPROVED** [review]

File: book/security.md (hunk 1 above)
Comment (pavpanchekha) [reply book/security.md:440]: Otherwise it sounds weird. Maybe "Although"?

Comment (chrishtr) [reply book/security.md:440]: Does it? Seems ok to me...

Comment (chrishtr) [reply book/security.md]: I updated the text to be phrased, better, WDYT?
